sum.py

```diff
@@ -6,13 +6,13 @@ def my_sum(a, b):
     Sums two variables and returns the result.
 
     Both parameters should be numbers.
-
     :param a: first addend
     :param b: second addend
     :return: sum of the addends
+    
     """
-    assert (is_number(a) and is_number(b)), 'Both addends should be numbers'
-    return a - b
+    assert (is_number(a) and is_number(b)), 
```

Comment:
Why did you remove the second expression from the [assert](https://docs.python.org/3.7/reference/simple_stmts.html?highlight=assert#the-assert-statement)? This change makes the code syntactically invalid.